packages/fxa-auth-db-mysql/lib/db/schema/patch-098-097.sql

```diff
@@ -0,0 +1,15 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
```

Comment:
We have a convention for these reverse migration files of commenting out every line, lest one get accidentally applied and e.g. production tables get dropped or something.

Reply:
Yeah, I was wondering about that - since I assumed the migration tool would auto-uncomment yet the reverse migration didn't seem to work :)